UI/src/components/Home.vue

```diff
@@ -4,8 +4,7 @@
             <h1 style="margin-bottom: 3em">{{ $t(`Welcome to LedgerSMB`) }}</h1>
             <div class="welcomed w1">
                 <h2>{{ $t(`What is LedgerSMB`) }}</h2>
-                <p>{{ $t(`LedgerSMB is a double entry accounting system offering a wide
-                  variety of related functionalities, such as`) }}</p>
+                <p>{{ $t(`i18n.home.ledgersmbIntro`) }}</p>
```

Comment:
Why did you switch to using keys as indexes to the translation instead of the original text?

Reply:
To remove white spaces imposed by line wrap in the translated strings.

Reply:
Ok. So, the Home page isn't a good example for in-page translation of HTML? I mean, it's probably better to have a  page like "Login" which has only a few fractions of text, instead of longer sections of explanatory text with external references. Maybe our "welcome" or Home page is best translated by storing HTML blobs on the server?